orm/model/ormlist/options.go

```diff
@@ -73,5 +36,23 @@ func Cursor(cursor CursorT) Option {
 	})
 }
 
+// Paginate paginates iterator output based on the provided page request.
+// The Iterator.PageRequest value on the returned iterator will be non-nil
+// after Iterator.Next() returns false when this option is provided.
+// Care should be taken when using Paginate together with Reverse and/or Cursor
+// and generally this should be avoided.
+func Paginate(pageRequest *queryv1beta1.PageRequest) Option {
+	return listinternal.FuncOption(func(options *listinternal.Options) {
+		if pageRequest.Reverse {
+			options.Reverse = !options.Reverse
```

Comment:
🤔

Reply:
So, no as the docs for `Reverse` indicate, multiple applications cancel each other out. This would allow for the default ordering for a query to be reverse, but then if PageRequest.Reverse is true, then this reverses reverse... If that makes sense.

Reply:
ah i see the comment now 👍🏻